modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFClaimForm/utils/submitForm.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {FormikHelpers} from 'formik';
+
+import MDFClaim from '../../../common/interfaces/mdfClaim';
+import createMDFClaimActivities from '../../../common/services/liferay/object/claim-activity/createMDFClaimActivities';
+import createMDFClaimActivityBudgets from '../../../common/services/liferay/object/claim-budgets/createMDFClaimActivityBudgets';
+import createMDFClaimDocuments from '../../../common/services/liferay/object/mdf-claim-documents/createMDFClaimDocuments';
+import createMDFClaim from '../../../common/services/liferay/object/mdf-claim/createMDFClaim';
+
+export default async function submitForm(
+	values: MDFClaim,
+	formikHelpers: Omit<FormikHelpers<MDFClaim>, 'setFieldValue'>,
+	_siteURL: string
```

Comment:
```suggestion
	siteURL: string
```
nao usamos o padrao de _